docs/resources/consul_cluster.md

```diff
@@ -51,6 +51,7 @@ resource "hcp_consul_cluster" "example" {
 ### Read-Only
 
 - `cloud_provider` (String) The provider where the HCP Consul cluster is located.
+- `cluster_state` (String) The state of the HCP Consul cluster.
```

Comment:
nit: similar to my comment about `snapshot_state` it feels like `cluster` in `cluster_state` is redundant. It might be preferable to call it just `state`. Then again there are a lot more fields here and its hard to see whether one name or the other is clearly better.